.travis.yml

```diff
@@ -10,6 +10,8 @@ osx: &osx
    language: generic
 matrix:
    include:
+      - <<: *linux
+        env: CONAN_GCC_VERSIONS=4.9 CONAN_DOCKER_IMAGE=lasote/conangcc49 CONAN_ARCHS=armv7hf
```

Comment:
the correct docker image is `lasote/conangcc49-armv7hf`

Reply:
I took it from here: https://github.com/conan-community/conan-openssl/blob/release/1.0.2o/.travis.yml#L28 maybe we need to check a lot of packages :/

Reply:
maybe CPT is appending the arch name as suffix :thinking:

Reply:
Yes, is getting from arch:

https://github.com/conan-io/conan-package-tools/blob/develop/cpt/packager.py#L497

thanks to Lasote!